docs/admins.rst

```diff
@@ -1,25 +1,122 @@
 Administration and Maintenance
 ==============================
 
-Setting up koji for container image builds
-------------------------------------------
+Configuring osbs-client
+-----------------------
+
+When submitting a new build request to OSBS as a user, this request is
+for an orchestrator build. When the orchestrator build wants to create
+worker builds it also does this through osbs-client.
+
+As a result there are two osbs.conf files to consider:
 
-The /etc/osbs.conf file used by the koji-containerbuild plugin must
-name secrets configured on the OpenShift orchestration cluster:
+- the one external to OSBS, for creating orchestrator builds, and
+- the one internal to OSBS, stored in a Kubernetes secret (named by
+  `client_config_secret`_) in the orchestrator cluster
+
+These can have the same content. The important features are discussed
+below.
+
+Can Orchestrate
```

Comment:
I'm wondering if this and `Token secrets` shouldn't be rather written as `can_orchestrate` and `token_secrets`, to match in style with `reactor_config_secret` and `client_config_secret`. (Or a similar change the other way around - reading the TOC feels inconsistent this way).

Reply:
How about 'Node selector' and 'Platform description' which don't apply to specific keys? I've left them as they are for now.

Reply:
Yes, I saw those, I think they are find as they are.